app/scripts/modules/core/src/application/application.component.ts

```diff
@@ -35,13 +35,15 @@ export class ApplicationController implements IComponentController {
     if (this.app.name.length > 20) {
       this.compactHeader = true;
     }
+    (window as any).application = this.app;
```

Comment:
Would it be better to namespace these? 

`spinnaker_application`
or
`spinnaker.application`